app/models/post.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  # Associations
+  belongs_to :user
+  has_many :comments
+  # Validations
+  validates :title, presence: true
```

Comment:
- [ ] **[OPTIONAL]** You can validate multipe attributes in one line if they all share the same validator:
`validates :title, :body, presence: true`